jinahub/rankers/CatboostRanker/tests/conftest.py

```diff
@@ -0,0 +1,128 @@
+import random
+
+import pytest
+import numpy as np
+from jina import Document, DocumentArray
+
+from ..catboost_ranker import CatboostRanker
+
+NUM_DOCS = 1000
+NUM_MATCHES = 5
+
+
+@pytest.fixture
+def ranker():
+    return CatboostRanker(
+        query_features=['brand', 'price'],
+        match_features=['brand', 'price'],
+        label='relevance',
+    )
+
+
+@pytest.fixture
+def ranker_with_weight():
+    return CatboostRanker(
+        query_features=['brand', 'price'],
+        match_features=['brand', 'price'],
+        label='relevance',
+        weight='weight',
+    )
+
+
+@pytest.fixture
+def relevances():
+    return np.random.uniform(0, 1, [1, NUM_DOCS]).flatten()
+
+
+@pytest.fixture
+def documents_to_train_stub_model(relevances):
+    """features: color, brand, price. Label relevance"""
+    # initial stub model, relevance purely dependent on brand, not price.
+    # brand relevance 5 > 4 > 3 > 2 > 1.
+    da = DocumentArray()
+    for relevance in relevances:
+        if 0.8 <= relevance <= 1:
+            brand = 5
+        elif 0.6 <= relevance < 0.8:
+            brand = 4
+        elif 0.4 <= relevance < 0.6:
+            brand = 3
+        elif 0.2 <= relevance < 0.4:
+            brand = 2
+        else:
+            brand = 1
```

Comment:
To make it easier to read, I would replace all this with `np.digitize` (you can do it also outside the loop)

Reply:
nice! learned something new @tadejsv